src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/AuthorMap/AuthorMap.xaml.cs

```diff
@@ -0,0 +1,497 @@
+                }
+                catch(Exception ex)
+                {
+                    MessageBox.Show("Unable to save map to ArcGIS Online: " + ex.Message);                    
+                }
+                finally
+                {
+                    // Hide the progress bar
+                    SaveProgressBar.Visibility = Visibility.Hidden;
+                }
+            }
+            else
+            {
+                // This is an update to the existing portal item
+                try
+                {
+                    // Show the progress bar so the user knows it's working
+                    SaveProgressBar.Visibility = Visibility.Visible;
+
+                    // This is not the initial save, call SaveAsync to save changes to the existing portal item
+                    await _myMap.SaveAsync();
+                    MessageBox.Show("Saved changes to '" + _myMap.PortalItem.Title + "'", "Updates Saved");
+                }
+                catch(Exception ex)
+                {
+                    MessageBox.Show("Unable to save map updates: " + ex.Message);
+                }
+                finally
+                {
+                    // Hide the progress bar
+                    SaveProgressBar.Visibility = Visibility.Hidden;
+                }
+            }
+        }
+
+        private void ClearMap(object sender, RoutedEventArgs e)
+        {
+            // Set the map to null
+            _myMap = null;
+
+            // Show a plain gray map in the map view
+            MyMapView.Map = new Map(Basemap.CreateLightGrayCanvas());
+        }
+        
+        private void UpdateViewExtentLabels()
+        {
+            // Get the current view point for the map view
+            Viewpoint currentViewpoint = MyMapView.GetCurrentViewpoint(ViewpointType.BoundingGeometry);
+            if (currentViewpoint == null) { return; }
+
+            // Get the current map extent (envelope) from the view point
+            Envelope currentExtent = currentViewpoint.TargetGeometry as Envelope;
+            // Project the current extent to geographic coordinates (longitude / latitude)
+            Envelope currentGeoExtent = GeometryEngine.Project(currentExtent, SpatialReferences.Wgs84) as Envelope;
+
+            // Fill the app text boxes with min / max longitude (x) and latitude (y) to four decimal places
+            XMinTextBox.Text = currentGeoExtent.XMin.ToString("0.####");
+            YMinTextBox.Text = currentGeoExtent.YMin.ToString("0.####");
+            XMaxTextBox.Text = currentGeoExtent.XMax.ToString("0.####");
+            YMaxTextBox.Text = currentGeoExtent.YMax.ToString("0.####");
+        }
+
+        #region OAuth helpers
+        private void UpdateAuthenticationManager()
+        {
+            // Register the server information with the AuthenticationManager
+            Esri.ArcGISRuntime.Security.ServerInfo portalServerInfo = new ServerInfo
```

Comment:
Is it necessary to fully qualify this statement with 'Esri.ArcGISRuntime.Security' or will the shorthand version work:
Ex: ServerInfo portalServerInfo = new ServerInfo